src/main/java/org/prebid/server/bidder/thirtythreeacross/ThirtyThreeAcrossBidder.java

```diff
@@ -39,16 +42,19 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-public class TtxBidder implements Bidder<BidRequest> {
+public class ThirtyThreeAcrossBidder implements Bidder<BidRequest> {
 
-    private static final TypeReference<ExtPrebid<?, ExtImpTtx>> TTX_EXT_TYPE_REFERENCE =
+    private static final TypeReference<ExtPrebid<?, ExtImpThirtyThreeAcross>> THIRTY_THREE_ACROSS_EXT_TYPE_REFERENCE =
             new TypeReference<>() {
             };
+    private static final ThirtyThreeAcrossExtTtxCaller PREBID_CALLER =
+            ThirtyThreeAcrossExtTtxCaller.of("Prebid-Server", "n/a");
```

Comment:
Hi, I am the maintainer from 33Across. Can I request the caller name passed as `Prebid-Server-Java` to differentiate requests from the Go adapter we already have? Thanks.